Rubberduck.Parsing/VBA/DeclarationResolving/ProjectsToResolveFromComProjectsSelector.cs

```diff
@@ -32,10 +37,28 @@ public void RefreshProjectsToResolveFromComProjectSelector()
             lock (_collectionLockObject)
             {
                 _projectsToResolveFromComProjects.Clear();
-                _projectsToResolveFromComProjects.UnionWith(_projectsProvider.LockedProjects().Select(tpl => tpl.ProjectId));
+                _projectsToResolveFromComProjects.UnionWith(LockedProjectIds());
+                _projectsToResolveFromComProjects.UnionWith(IgnoredProjectIds());
             }
         }
 
+        private IEnumerable<string> LockedProjectIds()
+        {
+            return _projectsProvider
+                .LockedProjects()
+                .Select(tpl => tpl.ProjectId);
+        }
+
+        private IEnumerable<string> IgnoredProjectIds()
+        {
+            var ignoredProjectFilenames = _ignoredProjectsSettingsProvider.Read().IgnoredProjectPaths;
+            return _projectsProvider
+                .Projects()
+                .Where(tpl => tpl.Project.TryGetFullPath(out var filename) 
+                                                && ignoredProjectFilenames.Contains(filename))
```

Comment:
I would not like to introduce such a capability in this PR since this will require a completely different mechanism to enter the patterns. None of the entry methods so far supports entering free text.